iphone/cli/commands/_build.js

```diff
@@ -5819,6 +5819,8 @@ iOSBuilder.prototype.copyResources = function copyResources(next) {
 			const contents = JSON.stringify(props);
 			if (!fs.existsSync(appPropsFile) || contents !== fs.readFileSync(appPropsFile).toString()) {
 				this.logger.debug(__('Writing %s', appPropsFile.cyan));
+
+				fs.existsSync(this.buildAssetsDir) || wrench.mkdirSyncRecursive(this.buildAssetsDir);
```

Comment:
Shouldn't you only do this if encryption is enabled?

```js
if (this.encryptJS && fs.existsSync(this.buildAssetsDir)) {
    wrench.mkdirSyncRecursive(this.buildAssetsDir);
}
```

Reply:
Good point, I had missed that we're not always working with buildAssetsDir, and looking at the flow `xcodeAppDir` will exist before this so I'll edit to only do this when we're encrypting